g2p/cli.py

```diff
@@ -95,7 +95,8 @@ def generate_mapping(in_lang, dummy, ipa, list_dummy, out_dir):
         eng_ipa = Mapping(in_lang="eng-ipa", out_lang="eng-arpabet")
         new_mapping = Mapping(in_lang=in_lang, out_lang=f"{in_lang}-ipa")
         click.echo(f"Writing English IPA mapping for {in_lang} to file")
-        create_mapping(new_mapping, eng_ipa, write_to_file=True, out_dir=out_dir)
+        create_mapping(new_mapping, eng_ipa,
+                       write_to_file=True, out_dir=out_dir)
```

Comment:
nitpicking here, but this is not our standard black formatting. I'm not picky about what format we use, but I prefer consistency...